common/djangoapps/student/views/management.py

```diff
@@ -504,14 +486,6 @@ def activate_account(request, key):
     """
     When link in activation e-mail is clicked
     """
-    # If request is in Studio call the appropriate view
-    if theming_helpers.get_project_root_name().lower() == u'cms':
-        monitoring_utils.set_custom_metric('student_activate_account', 'cms')
```

Comment:
**Note to Reviewers:** Thanks to @robrap's temporary metrics, we can verify that this CMS path is not needed.  Also, @robrap's parallel effort to remove Studio's own account login/registration pages help eliminate this path.

Reply:
I removed the decoupled commit related to the removal of `student_activate_account`.  That should unblock this PR.